WorkloadManagementSystem/DB/JobDB.py

```diff
@@ -1486,10 +1489,31 @@ def getSiteMask( self, siteState = 'Active' ):
     return S_OK( siteList )
 
 #############################################################################
-  def getSiteMaskStatus( self ):
+  def getSiteMaskStatus( self, sites = None ):
     """ Get the currently site mask status
     """
-    cmd = "SELECT Site,Status FROM SiteMask"
+    if isinstance(sites, list):
+
+      cmd = "SELECT Site, Status FROM SiteMask WHERE"
```

Comment:
I think I don't understand what you are trying to say, can you please elaborate? 
`cmd` contracts an SQL query, why should it be changed? Also, is `('site1', 'site2')` considered to be the `sites` list?